tests/test_hash.py

```diff
@@ -1,77 +1,43 @@
 import pytest
```

Comment:
id like to see a `test_abi` just to kinda show-case how to use those features, it doesnt have to be comprehensive